src/Telemetry/Exit_Interview_Template.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace StellarWP\Telemetry;
+
+use StellarWP\Telemetry\Contracts\Template;
+
+class Exit_Interview_Template implements Template {
+
+	protected function get_args(): array {
+		return apply_filters( 'stellarwp/telemetry/exit_interview_args', [
```

Comment:
We need the plugin slug in the filter so each plugin and update the logo and/or text individually.